extensions/wikia/PhalanxII/hooks/PhalanxHooks.class.php

```diff
@@ -43,8 +41,9 @@ static public function loadLinks( $id, $nt, &$links ) {
 	 * @param $text string content to check for spam
 	 * @param $typeId int block type (see Phalanx::TYPE_* constants)
 	 * @param $blockData array array to be provided with matching block details (pass as a reference)
-	 * @return boolean spam check result
+	 * @return bool spam check result
 	 *
+	 * @throws WikiaException
 	 * @author macbre
 	 */
 	static public function onSpamFilterCheck( $text, $typeId, &$blockData ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$blockData". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app